service/src/test/java/bio/terra/profile/db/ProfileChangeLogDaoTest.java

```diff
@@ -0,0 +1,83 @@
+package bio.terra.profile.db;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import bio.terra.profile.common.BaseSpringUnitTest;
+import bio.terra.profile.model.ChangeLogEntry;
+import bio.terra.profile.model.CloudPlatform;
+import bio.terra.profile.service.profile.model.BillingProfile;
+import java.sql.Date;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class ProfileChangeLogDaoTest extends BaseSpringUnitTest {
+
+  @Autowired ProfileChangeLogDao dao;
+
+  @Test
+  void changeLogCreatesEntryForCreate() {
+    var user = "user@test.com";
+    var profileCreateTime = Instant.now().minus(10, ChronoUnit.SECONDS);
+    var profile =
+        new BillingProfile(
+            UUID.randomUUID(),
+            "profile name",
+            "profile description",
+            "direct",
+            CloudPlatform.AZURE,
+            Optional.empty(),
+            Optional.of(UUID.randomUUID()),
+            Optional.of(UUID.randomUUID()),
+            Optional.of("mrg_id"),
+            profileCreateTime,
+            profileCreateTime,
+            user);
+    var recordId = dao.recordProfileCreate(profile);
+    var records = dao.getChangesByProfile(profile.id());
```

Comment:
Formatting nit, linebreaks for the usual "arrange/act/assert" sections would help readability.